packages/eas-cli/src/build/build.ts

```diff
@@ -140,12 +140,34 @@ function handleBuildRequestError(error: any, platform: Platform): never {
     error?.graphQLErrors?.[0]?.extensions?.errorCode === 'EAS_BUILD_DOWN_FOR_MAINTENANCE'
   ) {
     Log.error(
-      'EAS Build is down for maintenance. Try again later. Check https://status.expo.dev/ for updates.'
+      `EAS Build is down for maintenance. Try again later. Check ${link(
+        'https://status.expo.dev/'
+      )} for updates.`
     );
     throw new Error('Build request failed.');
   } else if (error?.graphQLErrors?.[0]?.extensions?.errorCode === 'EAS_BUILD_FREE_TIER_DISABLED') {
     Log.error(
-      'EAS Build free tier is temporarily disabled. Try again later. Check https://status.expo.dev/ for updates.'
+      `EAS Build free tier is temporarily disabled. Try again later. Check ${link(
+        'https://status.expo.dev/'
+      )} for updates. ${learnMore('https://expo.fyi/eas-build-queues')}`
+    );
+    throw new Error('Build request failed.');
+  } else if (
+    error?.graphQLErrors?.[0]?.extensions?.errorCode === 'EAS_BUILD_FREE_TIER_DISABLED_IOS'
+  ) {
+    Log.error(
+      `EAS Build free tier is temporarily disabled for iOS. Try again later. Check ${link(
+        'https://status.expo.dev/'
+      )} for updates. ${learnMore('https://expo.fyi/eas-build-queues')}`
+    );
+    throw new Error('Build request failed.');
+  } else if (
+    error?.graphQLErrors?.[0]?.extensions?.errorCode === 'EAS_BUILD_FREE_TIER_DISABLED_ANDROID'
+  ) {
+    Log.error(
+      `EAS Build free tier is temporarily disabled for Android. Try again later. Check ${link(
+        'https://status.expo.dev/'
+      )} for updates. ${learnMore('https://expo.fyi/eas-build-queues')}`
```

Comment:
@brentvatne Are these error messages correct? As far as I understand, these are the user-facing ones 🙂

Reply:
I'm not 100% sold on the wording, as the situation is that we stop accepting new builds, not that we've disabled EAS Build, as all queued builds will be processed, right?

Second, would we always have a Statuspage incident when using these kill switches? The error message makes that a requirement. And how would this interact with the other feature we added for showing the user any statuspage incident if there is one?

Reply:
I would drop the status page part.